### PR TITLE
feat: force to install dependencies

### PR DESCRIPTION
**Pull request by Ian2012**
**Merge checklist:**
Check off if complete *or* not applicable:
- [ ] Version bumped
- [ ] Changelog record added
- [ ] Documentation updated (not only docstrings)
- [ ] Fixup commits are squashed away
- [ ] Unit tests added/updated
- [ ] Manual testing instructions provided
- [ ] Noted any: Concerns, dependencies, migration issues, deadlines, tickets

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,74 +1,64 @@
...
+To deploy this plugin in your Open edX platform within the Unidigital project,
+execute the following command:
 
-Deploying
-*********
+.. code-block:: bash
 
-TODO: How can a new user go about deploying this component? Is it just a few
-commands? Is there a larger how-to that should be linked here?
+    pip install openedx-unidigital
```
Comment (mariajgrimaldi) [inline README.rst]: Since this is not available in pypi, can we modify this command to use a github source?

File: README.rst [hunk 2]
```diff
@@ -1,74 +1,64 @@
...
+=======
 
-Our real-time conversations are on Slack. You can request a `Slack
-invitation`_, then join our `community Slack workspace`_.
+- `Limesurvey XBlock <https://github.com/eduNEXT/xblock-limesurvey>`_: An XBlock for embedding Limesurvey surveys within a course.
+- `Mindmap XBlock <https://github.com/eduNEXT/xblock-mindmap>`_: An XBlock for embedding Mindmaps within a course.
+- `Files Manager XBlock <https://github.com/eduNEXT/xblock-filesmanager>`_: An XBlock for sharing files with students within a course.
+- `H5P XBlock <https://github.com/eduNEXT/h5pxblock>`_: An XBlock for embedding H5P content within a course.
+- `Feedback XBlock <https://github.com/eduNEXT/FeedbackXBlock>`_: An XBlock for embedding a Feedback form within a course.
+- `edx-ora2 <https://github.com/eduNEXT/edx-ora2>`_: An XBlock for embedding ORA2 content within a course.
```
Comment (mariajgrimaldi) [inline README.rst]: what about the imagesgallery xblock?

**Comment by mariajgrimaldi** [comment]
can we document the process of adding new packages? which ones should we include here?

File: requirements/base.in [hunk 3]
```diff
@@ -5,3 +5,23 @@ Django             # Web application framework
 
 
 openedx-atlas
+# Aspects requirements
+openedx-event-sink-clickhouse
+edx-event-routing-backends
+edx-completion>=4.4.0
+openedx-events>=9.2.0
+# Plugins
```
Comment (mariajgrimaldi) [inline requirements/base.in]:
```suggestion
# eduNEXT Plugins
```

File: requirements/base.in [hunk 4]
```diff
@@ -5,3 +5,23 @@ Django             # Web application framework
...
+xblock-mindmap @ git+https://github.com/eduNEXT/xblock-mindmap@v0.9.0
+xblock-imagesgallery @ git+https://github.com/eduNEXT/xblock-imagesgallery.git@v0.4.2
+xblock-filesmanager @ git+https://github.com/eduNEXT/xblock-filesmanager.git@v0.4.0
+platform-plugin-forum-email-notifier @ git+https://github.com/edunext/platform-plugin-forum-email-notifier@v0.3.2
+platform-plugin-superset @ git+https://github.com/edunext/platform-plugin-superset@v0.1.0
+# git+https://github.com/eduNEXT/platform-plugin-ontask
+# git+https://github.com/eduNEXT/platform-plugin-groups
+# git+https://github.com/eduNEXT/platform-plugin-oncampus
+# git+https://github.com/eduNEXT/platform-plugin-turnitin
+# Official plugins
```
Comment (mariajgrimaldi) [inline requirements/base.in]:
```suggestion
# Community plugins
```

File: tests/test_models.py [hunk 5]
```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env python
```
Comment (mariajgrimaldi) [inline tests/test_models.py:1]: do we need this file?

File: test_settings.py [hunk 6]
```diff
@@ -39,8 +39,6 @@ def root(*args):
     root("openedx_unidigital", "conf", "locale"),
```
Comment (mariajgrimaldi) [inline test_settings.py:39]: do we need this file?

File: README.rst (hunk 1 above)
Comment (Ian2012) [reply README.rst]: Sure

File: tests/test_models.py (hunk 5 above)
Comment (Ian2012) [reply tests/test_models.py:1]: yes, if there isn't a single test CI will fail

File: test_settings.py (hunk 6 above)
Comment (Ian2012) [reply test_settings.py:39]: Yes, it's required for the tests

**Review by mariajgrimaldi — APPROVED** [review]

**Pull request merged**